app/assets/javascripts/govuk_publishing_components/lib/cookie-functions.js

```diff
@@ -82,20 +101,47 @@
     window.GOVUK.setCookie('cookie_policy', JSON.stringify(cookieConsentJSON))
   }
 
-  window.GOVUK.setCookie = function (name, value, options) {
-    if (typeof options === 'undefined') {
-      options = {}
+  window.GOVUK.checkConsentCookie = function (cookieName, cookieValue) {
+    var currentConsentCookie = window.GOVUK.getCookie('cookie_policy')
+
+    // If we're setting the consent cookie OR deleting a cookie, allow by default
+    if (cookieName === 'cookie_policy' || (cookieValue === null || false)) {
```

Comment:
The `|| false` is redundant here. I think what you may have meant is this:

```suggestion
    if (cookieName === 'cookie_policy' || (cookieValue === null || cookieValue === false)) {
```